### ISSUE TITLE
Installation with npm fails

### ISSUE THREAD

**Issue by gish** [nordnet/grid#1]
I'm getting this error when running an upgrade locally. Our Jenkins runs npm, which also fails after the installation step.

```bash
$ yarn upgrade @nordnet/grid@^1.0.1
yarn upgrade v0.24.5
[1/4] 🔍  Resolving packages...
[2/4] 🚚  Fetching packages...
[3/4] 🔗  Linking dependencies...
warning "jss-isolate@2.0.1" has incorrect peer dependency "jss@^6.0.0".
[4/4] 📃  Rebuilding all packages...
[1/3] ⠂ fsevents
[2/3] ⠂ husky
[3/3] ⠂ @nordnet/grid
[-/3] ⠂ waiting...
error /Users/erikhedberg/dev/news-component/node_modules/@nordnet/grid: Command failed.
Exit code: 127
Command: sh
Arguments: -c onpm docs install
Directory: /Users/erikhedberg/dev/news-component/node_modules/@nordnet/grid
Output:
sh: onpm: command not found
info Visit https://yarnpkg.com/en/docs/cli/upgrade for documentation about this command.
```

**Comment by bstream** [comment]
I think it is because of the fix introduced in 5d5d2f542450f5493ac3ac4f560de995a03498b5. Will make a PR reverting that!

**Comment by iamstarkov** [comment]
fixed
